### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by chirchir12**
Here is my Apple Store Solo Project completely accomplished using linear gradient and background images, background positions and background size.
The layout has been created using Flex box and CSS grid

### CODE REVIEW

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

File: index.html
Comment (christianotieno) [inline index.html]: ## [REQUIRED] - CHANGE - Elements to links.
- [ ] These four elements should be links, let's add them inside of the 'a' tags.

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @chirchir12  :wave: 
-
You have done a good with this one! :tada:  Keep it up. We are almost there! We just need to take care of some few things before we approve this. Kindly see the review comments below.
##
### NOTES: I am conducting this review with a (1920 x 1080) screen display resolution and your final work isn't rendering properly as it should. 
- This is what I am seeing:
clone

- - [ ] Fix the screen content of the page to only fill the screen's maximum ViewHeight (Which is 100%). This should remove the scroll that is currently present. 
- - [x] Change the four image sections at the bottom to be links.
##
- That should be all. Please make the specified changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @chirchir12 👋

You've done quite well on this.👌🎉🎉
Working with previous reviews to get to this point, is awesome.👏

However, there are a few changes to be made:👇

- [ ] Required - There are a few issues on your navbar. Kindly fix them:
**This is yours👇**
ice_screenshot_20200408-000011

**This is the way it should be**
ice_screenshot_20200408-000101

* Add a search icon to the search input field
* Add a border-radius to the navbar.

- [ ] Required - Also kindly change the icon you used here.
ice_screenshot_20200407-235948

**THIS IS THE WAY IT SHOULD BE**
ice_screenshot_20200408-000118
Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by chirchir12** [comment]
Hello I fixed the icon in search bar 
I so replaced the square icon with circle
Review it again 
Regards

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @chirchir12 👋

You've done quite well on this, working on the previous review, and fixing the linting issues in one go. You are on a roll.👏

However, on parsing your HTML through an HTML validator tool I noticed a few errors, kindly fix this and request another review. Use this [tool](https://validator.w3.org/nu/#file) to validate your HTML.

Kindly fix these issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by chirchir12** [comment]
Thank you for the review. I fixed unclosed a tag. Kindly review it again. I appreciate all the reviews.

**Comment by sumancrest0001** [comment]
**STATUS: CHANGES REQUIRED**

Anwesome job @chirchir12 :clap:  :clap: 
You have made all the changes requested by the previous TSE. But we have to tick one requirement before moving ahead. In this project, we have to use either **float and flex** or **float and grid** technique for the alignment of items. Currently, you are using flex only. Please use float also and submit for another review.
Need help? Message me on slack **Suman Shrestha**

**Comment by chirchir12** [comment]
Hello, 
I managed to makes the changes requested and used floats in my footer. Kindly review

**Comment by thsvr** [comment]
### **STATUS: PROJECT APPROVED** :fireworks:  :fireworks:  :fireworks:  :fireworks:  :fireworks: 

Hi @chirchir12 :smile: :smile: 

Congrats for the hard work :muscle: 
You did a great job and your project was approved :1st_place_medal: :1st_place_medal: 

Before merge it into master, can you please do the next simple 2 changes? 
### **NO NEED TO SUBMIT AGAIN FOR ANOTHER REVIEW, OK?** :+1: :smile_cat: 

:v: Can you please fix the title of your readme? The title is ''new-york-times-clone''.

:v: Can you please update the live version to show a hyperlink? In your readme it is showing a link. The way to do it is just as the usual `<a href="">Live version</a>`


_**SUGGESTION FOR THE NEXT PROJECTS:**_

:v: You can create a feature branch with a more descriptive name on the next projects, like ''main-page'' or ''apple-page'' instead of ''feature', for example. It will be more descriptive :smile_cat: :+1: 




If you have any questions, don't hesitate to contact me through slack [@thaís Vieira Azevedo](url)


### **Happy coding and good luck on the next project** :computer: 

_Project reviewed by Thaís Vieira (TSE)_

**Comment by chirchir12** [comment]
updated the changes as requested

**Pull request merged**
